browser/src/map/handler/Map.Keyboard.js

```diff
@@ -140,9 +140,37 @@ L.Map.Keyboard = L.Handler.extend({
 	},
 
 	keyCodes: {
+
+		BACKSPACE:8,
+		TAB:      9,
+		RETURN:   13, 
+		SPACE :   32,
 		pageUp:   33,
 		pageDown: 34,
-		enter:    13,
+		SHIFT:    16, // shift		: UNKOWN
```

Comment:
@eszkadev I was talking about this 16, Is this okay to put it like this?

Reply:
I see no problem with that (assuming value is correct) :)
But why should it be also UNKNOWN?

Reply:
I got the value from `16  : null, // shift` which is present in the `keymap` dictionary of the same file

Reply:
'shift: unknown' comment doesn't appy to this enum, it was about UNO codes - it's something different.

Reply:
please do not look at UNO key codes, that is something completly different, only for our server. here we want to make names for JS key codes.